apps/web/src/components/EventPlanList/EventPlanList.tsx

```diff
@@ -8,16 +6,14 @@ import {
   EventPlanFinalizedFlag,
 } from '../../interfaces';
 
-import './EventPlanList.css';
-
 interface EventPlanListProps {
   elementId: string;
   eventPlans: (EventPlanInfo & { eventPlanId: EventPlanId } & {
     isFinalized: EventPlanFinalizedFlag;
   })[];
 }
 
-function _sortEventPlans(
+function sortEventPlans(
   eventPlans: (EventPlanInfo & { eventPlanId: EventPlanId } & {
```

Comment:
This typing is kinda hard to understand -- my fault trying to make certain interfaces reusable, this was before reworking the functions api, back when we were only using firebase rules to control read / write. 

Will clean up the types and interfaces.

Reply:
Okay awesome, are you going to making the changes that correspond to those types/interfaces?

Reply:
Yes, just going thru rest of file changes to fix anything minor before doing so